includes/machine-learning-service-inference-config.md

```diff
@@ -11,15 +11,16 @@ The entries in the `inferenceconfig.json` document map to the parameters for the
+| `condaFile` | `conda_file` | Optional. Path to a local file that contains a Conda environment definition to use for the image.  If `environment` is set, this gets ignored. |
+| `extraDockerFileSteps` | `extra_docker_file_steps` | Optional. Path to a local file that contains additional Docker steps to run when setting up the image.  If `environment` is set, this gets ignored.|
 | `sourceDirectory` | `source_directory` | Optional. Path to folders that contain all files to create the image. |
-| `enableGpu` | `enable_gpu` | Optional. Whether to enable GPU support in the image. The GPU image must be used on an Azure service, like Azure Container Instances, Azure Machine Learning Compute, Azure Virtual Machines, and Azure Kubernetes Service. The default is False. |
-| `baseImage` | `base_image` | Optional. Custom image to be used as a base image. If no base image is provided, the image will be based on the provided runtime parameter. |
-| `baseImageRegistry` | `base_image_registry` | Optional. Image registry that contains the base image. |
-| `cudaVersion` | `cuda_version` | Optional. Version of CUDA to install for images that need GPU support. The GPU image must be used on an Azure service, like Azure Container Instances, Azure Machine Learning Compute, Azure Virtual Machines, and Azure Kubernetes Service. Supported versions are 9.0, 9.1, and 10.0. If `enable_gpu` is set, the default is 9.1. |
-| `description` | `description` | A description for the image. |
+| `enableGpu` | `enable_gpu` | Optional. Whether to enable GPU support in the image. The GPU image must be used on an Azure service, like Azure Container Instances, Azure Machine Learning Compute, Azure Virtual Machines, and Azure Kubernetes Service. The default is False. If `environment` is set, this gets ignored.|
+| `baseImage` | `base_image` | Optional. Custom image to be used as a base image. If no base image is provided, the image will be based on the provided runtime parameter. If `environment` is set, this gets ignored. |
+| `baseImageRegistry` | `base_image_registry` | Optional. Image registry that contains the base image. If `environment` is set, this gets ignored.|
+| `cudaVersion` | `cuda_version` | Optional. Version of CUDA to install for images that need GPU support. The GPU image must be used on an Azure service, like Azure Container Instances, Azure Machine Learning Compute, Azure Virtual Machines, and Azure Kubernetes Service. Supported versions are 9.0, 9.1, and 10.0. If `enable_gpu` is set, the default is 9.1. If `environment` is set, this gets ignored. |
+| `description` | `description` | A description for the image. If `environment` is set, this gets ignored.  |
+| `environment` | `environment` | Optional.  Azure Machine Learning [environment](https://docs.microsoft.com/python/api/azureml-core/azureml.core.environment.environment?view=azure-ml-py).|
```

Comment:
Can we just use the description from https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.model.inferenceconfig?view=azure-ml-py for these parameters? They come from python comments directly.